packages/pyright-internal/src/localization/simplified.nls.de.json

```diff
@@ -0,0 +1,93 @@
+{
+    "Diagnostic": {
+        "argAssignment": "Argument stimmt nicht mit Parametertyp überein",
+        "argAssignmentFunction": "Argument stimmt nicht mit dem Parametertyp in Funktion \"{functionName} \" überein",
+        "argAssignmentParam": "Argument stimmt nicht mit Parametertyp für Parameter \"{paramName} \" überein",
```

Comment:
`functionName` and `paramName` have an extra space after the curly brace in both cases.

Reply:
Thanks - I'll fix this and do another update.